src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.cs

```diff
@@ -158,5 +160,367 @@ public static void ImportSubjectPublicKeyInfo_NullSource()
             AssertExtensions.Throws<ArgumentNullException>("source", static () =>
                 MLKem.ImportSubjectPublicKeyInfo((byte[])null));
         }
+
+        [Fact]
+        public static void ImportPkcs8PrivateKey_NullSource()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("source", static () =>
+                MLKem.ImportPkcs8PrivateKey((byte[])null));
+        }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public static void ImportPkcs8PrivateKey_WrongAlgorithm(bool useSpanImport)
```

Comment:
I don't recall seeing a test that had the correct OID but incorrectly encoded the parameters as NULL (or, well, anything) instead of [absent].